x-pack/plugins/security_solution/server/features.ts

```diff
@@ -116,16 +116,17 @@ export const getKibanaPrivilegesFeaturePrivileges = (ruleTypes: string[]): Kiban
   subFeatures: [],
   privileges: {
     all: {
-      app: [APP_ID, 'kibana'],
+      app: [APP_ID, 'csp', 'kibana'],
       catalogue: [APP_ID],
-      api: [APP_ID, 'lists-all', 'lists-read', 'lists-summary', 'rac'],
+      api: [APP_ID, 'lists-all', 'lists-read', 'lists-summary', 'rac', 'csp-all'],
```

Comment:
Please rename `csp-all` to `cloud-security-posture-all` - `csp` has semantic meaning in Kibana

Reply:
will do. 

CSP was a known acronym before we picked our name and we did it anyway. now we have many usages of `Csp` or `CSP` in our plugin. i think we should keep discussing this and come up with another acronym, because a long string isn't as convenient it many cases.